src/pprint/js_pprint.cc

```diff
@@ -684,6 +684,7 @@ counted_t<const document_t> render_as_javascript(const ql::raw_term_t &t) {
 // - `should_continue_string` should add the new Term to the list if
 //   it should be used in a string-of-dotted-expressions context.  So
 //   like `r.foo(1).bar(2).baz(4)` instead of `r.eq(1, 2)`.
+//   ^^ What does this mean??
```

Comment:
Actually I'm wrong about the `r.eq()` example. It appears the pretty-printer always prefers `r.eq(args...)`, but I think the `.db()` one holds.